zgrab_schema.py

```diff
@@ -761,6 +761,11 @@
     "raw":Binary(),
 })
 
+golang_crypto_param = SubRecord({
+    "value":Binary(),
+    "length":Integer()
```

Comment:
The ZGrab schema doesn't appear to contain integer sized or make the `Binary`/`IndexedBinary` distinction, while the ZTag schema _does_ contain that info. I continued with that convention here, though I'm happy to adjust as needed.